GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -1,14 +1,64 @@
+/**
```

Comment:
Can you add `// @ts-check` so that the typing is checked?

Reply:
No, it raises tons of errors because it doesn't like optional arguments/the way they are treated. Maybe I can make the typing work in another PR but for now having just the documentation is enough I think.

Reply:
Ok fair enough!